Update config instructions (#56)
LycheeOrg/LycheeOrg.github.io: README.md

- [Install Python 3.x](https://www.python.org/downloads/)
- [Install pip](https://pip.pypa.io/en/stable/installing/)
- Install dependencies:

```sh
pip install -r requirements.txt
```

# Generating

> Just run `make` which generates the files in to `build/`. You don't have to commit them.
>  

Reviewer: ```suggestion
Run `make` to generate the html, which can then be found in `build/`.

```

Maybe? I don't think we need to explicitly mention committing as `build/*` is in `.gitignore`.
